resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/kubernetes/KubernetesClusterSchedulerBackend.scala

```diff
@@ -521,3 +597,12 @@ private object KubernetesClusterSchedulerBackend {
   private val DEFAULT_STATIC_PORT = 10000
   private val EXECUTOR_ID_COUNTER = new AtomicLong(0L)
 }
+
+// These case classes model K8s node affinity syntax for
```

Comment:
Use `/** ... */` style comments for something like this, particularly if we're embedding an `@see` tag below.